README.md

```diff
@@ -105,155 +151,153 @@ Ensure that the following roles are added to your service account:
+* `roles/cloudsql.admin`
 
-Then create the secret using the given `creds.json` file:
+It is convenient to assign roles with `gcloud` CLI, e.g.
 
 ```console
-kubectl -n upbound-system create secret generic gcp-creds --from-file=key=./creds.json
+gcloud projects add-iam-policy-binding --role="$ROLE" $PROJECT_ID --member "serviceAccount:$SA"
 ```
 
-Create the `ProviderConfig`, ensuring to set the `projectID` to your specific GCP project:
-
-```console
-kubectl apply -f examples/provider-default-gcp.yaml
+# Configure the GCP Provider to use the secret:
+kubectl apply -f examples/gcp-default-provider.yaml
 ```
 
-#### Invite App Teams to you Organization in Upbound Cloud
-
-1. Create a Team `team1`.
-1. Invite app team members and grant access to `Control Planes` and `Repositories`.
+See [provider-gcp docs](https://marketplace.upbound.io/providers/upbound/provider-gcp/latest/docs/configuration) for more detailed configuration options.
 
-### App Dev/Ops: Consume the infrastructure you need using kubectl
+## Using the GCP reference platform
 
-#### Join your Organization in Upbound Cloud
+🎉 Congratulations. You have just installed your first Crossplane powered
+platform!
 
-1. **Join** your [Upbound Cloud](https://cloud.upbound.io/register)
-   `Organization`
-1. Verify access to your team `Control Planes` and Registries
+Application developers can now use the platform to request resources which than
+will provisioned in GCP. This would usually done by bundling a claim as part of
+the application code. In our example here we simply create the claims directly:
 
-#### Provision a Kubernetes cluster using kubectl
-
-1. Provision a `Cluster` resource (claim for `CompositeCluster`) provided by the platform `Configuration`:
+Create a custom defined cluster:
+```console
+kubectl apply -f examples/cluster-claim.yaml
+```
 
+Create a custom defined database:
 ```console
-kubectl -n upbound-system apply -f examples/cluster.yaml
+kubectl apply -f examples/postgres-claim.yaml
 ```
 
-1. View status / details of the managed resources created for your claim:
+You can verify status by inspecting the claims, composites and managed
+resources:
 
 ```console
-kubectl get managed
+kubectl get claim,composite,managed
 ```
 
-1. Check status of your claim:
+If you see an error about the `compute.globalAddresses.list` permission for the project, try running the following gcloud command:
+```
+gcloud projects add-iam-policy-binding HOST_PROJECT_NAME \
+    --member=serviceAccount:service-HOST_PROJECT_NUMBER@service-networking.iam.gserviceaccount.com \
+    --role=roles/servicenetworking.serviceAgent
+```
+
+This recommendation is an excerpt from [the official GCP
+documentation](https://cloud.google.com/sql/docs/postgres/configure-private-services-access)
+
+To delete the provisioned resources, you would simply delete the claims again:
 
 ```console
-kubectl -n upbound-system get cluster
+kubectl delete -f examples/cluster-claim.yaml,examples/postgres-claim.yaml
```

Comment:
I got caught with hanging resources that required manual deletion (the helm release and SQL user). Perhaps we should document the scripts.